src/core/components/ServerHtml/index.js

```diff
@@ -114,13 +114,13 @@ export default class ServerHtml extends Component {
   }
 
   renderStyles() {
-    const { _config, chunkExtractor } = this.props;
+    const { chunkExtractor } = this.props;
 
     return chunkExtractor
       .getMainAssets('style')
       .filter(
         // We render the main bundle with `getScript()`, so we skip it here.
-        (asset) => !_config.get('validAppNames').includes(asset.chunk),
+        (asset) => asset.chunk !== 'amo',
```

Comment:
maybe we should use `_config.get('appName')`, wdyt?

Reply:
Sure, that makes sense.